CIP-DappConnector/CIP-dapp-connector.md

```diff
@@ -243,27 +245,6 @@ Errors: `APIError`, `TxSendError`
 
 As wallets should already have this ability, we allow dApps to request that a transaction be sent through it. If the wallet accepts the transaction and tries to send it, it shall return the transaction id for the dApp to track. The wallet is free to return the `TxSendError` with code `Refused` if they do not wish to send it, or `Failure` if there was an error in sending it (e.g. preliminary checks failed on signatures).
 
-## Events
-
-In addition to the API methods that are actively called, the connector also must emit the following events. All methods events are required to be implemented.
-
-TODO: event emission method? Possible methods:
-
-1) Emitted event via `window.addEventListener(eventName , e => void)`
-2) Emitted message via `window.postMessage({eventName: string}, ...)`
-3) Some kind of callback registration i.e. `wallet.onDisconnect(() => void)` or `wallet.onEvent(eventName => void)`
-4) A combination of the two (event/message but with callback on `wallet` object as well
-5) Other?
-
-### wallet_disconnected
-
-Emitted when the user disconnects (not just changes) their current account from the page. This means that all `api` methods will return with an `APIError.Refused` error and a new `api` object must be obtained from `wallet.enable()` to continue to interact with the user's wallet.
-
-### account_changed
-
-Emitted when the user changes accounts (i.e. different root key pair and/or network id). The same `api` object will continue to work but will now return results based on the new account. After this event is received dApps should check `api.getNetworkId()` as changing accounts can also change the network.
-
-
 # Implementations
 
-TODO: link to Yoroi's implementation once available
+[nami-wallet](https://github.com/Berry-Pool/nami-wallet/blob/master/src/pages/Content/injected.js)
```

Comment:
CIP1 does mention there should be tests for implementations but I'm not sure how you'd test that for code injection so maybe it's fine. nami-wallet does also implement part of the events API which is removed from this CIP now as well.

Reply:
@rooooooooob The tests refer to changes on core aspects of the protocol one would propose to change - this being a connector, no tests required. From here once merged, a "Path to Active" proposal / addition to your PR, so it can be moved to 'Proposed' or 'Active' (ex: "I propose this be moved to 'Active' when x dapps have demonstrated integrations aligned with this CIP")